packages/zowe-explorer/CHANGELOG.md

```diff
@@ -984,3 +984,7 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 ## 0.5.0
 
 - Initial release
+
+## TBD Release
+
+- Added the Issue UNIX Commands feature. [#1326](https://github.com/zowe/vscode-extension-for-zowe/issues/1326)
```

Comment:
can we get this changelog moved to top of file please

Reply:
Yes moved to the top